decidim-templates/spec/system/admin/admin_manages_proposal_answer_templates_spec.rb

```diff
@@ -0,0 +1,151 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Admin manages proposal answer templates", type: :system do
+  let!(:organization) { create :organization }
+  let!(:user) { create :user, :confirmed, organization: organization }
+
+  before do
+    switch_to_host(organization.host)
+    login_as user, scope: :user
+    visit decidim_admin_templates.proposal_answer_templates_path
+  end
+
+  describe "listing templates" do
+    let!(:template) { create(:template, :proposal_answer, organization: organization) }
+
+    before do
+      visit decidim_admin_templates.proposal_answer_templates_path
+    end
+
+    it "shows a table with the templates info" do
+      within ".questionnaire-templates" do
+        expect(page).to have_i18n_content(template.name)
+        expect(page).to have_i18n_content("Global scope")
+      end
+    end
+  end
+
+  describe "creating a questionnaire_template" do
```

Comment:
```suggestion
  describe "creating a proposal_answer_template" do
```
Is not a questionnaire right?